shell.nix

```diff
@@ -13,6 +13,10 @@ with pkgs;
         metacraft-labs.cosmos-theta-testnet
         metacraft-labs.circom
         metacraft-labs.circ
+
+        # Elrond
+        metacraft-labs.erdpy
```

Comment:
Let's also start a new practice of providing a short description of all provided packages in the README of nix-blockchain-development. One or two sentences with a link to the upstream documentation would be sufficient.

Reply:
Should I add descriptions for all packages, or just the ones i've written?
Cause I feel like it might be a better to idea to give the descriptions in a separate PR.

Reply:
I'll be happy if you can do it for all packages, but if it's too difficult to gain knowledge about some of them, just placing placeholder entries without a description would be OK too. You are right that it can be done in a separate PR, but I wish it won't be postponed for too long. Ideally, you would do it before you start work on the next protocols.